internal/printer/printer_test.go

```diff
@@ -372,6 +374,12 @@ const name = "world";
 				imports:     "",
 				frontmatter: []string{},
 				styles:      []string{},
+				resources: []string{
+					`{'rel':'icon','href':'/favicon.ico'}`,
+					`{'rel':'icon','href':'/favicon.svg','type':'image/svg+xml'}`,
+					`{'rel':'apple-touch-icon','href':'/apple-touch-icon.png'}`,
+					`{'rel':'stylesheet','href':'css/styles.css?v=1.0'}`,
```

Comment:
Adds `resources[]` as a new optional property in tests. This will make `metadata` less annoying to edit not being a super long string.

Reply:
I really like that. I’ll try to find a balanced approach between Go structs and strings (I originally started with map slices and having deeply-typed Go structures in tests are a little annoying IMO).

Reply:
Updated!